_posts/2022-01-20-saying-no.md

```diff
@@ -0,0 +1,43 @@
+I've developed a bit of a reputation inside and outside work as being pretty good with "saying no" and setting boundaries. In this post, I will help you do so, too.
+
+### Why say "no"
+
+I've been consuming a lot of Brené Brown's [podcasts](https://brenebrown.com/podcasts/) and read [one of her books](https://brenebrown.com/book/atlas-of-the-heart/) recently, and they've got me thinking about what's at the root of being able to say "no" regularly and well.
+
+[![BRAVING Trust]({{ '/images/a/braving.jpg' | absolute_url }})](https://www.linkedin.com/pulse/braving-trust-brené-brown/)
+
+Brown uses the [BRAVING acronym](https://www.linkedin.com/pulse/braving-trust-brené-brown/) to refer to the elements of Trust. Out of these, the ones that jump out to me for saying no are boundaries, reliability and integrity. To meaningfully trust others on these axes, you cannot be someone who says yes to every request. In doing so, you maintain no boundaries, professionally or personally, and will not be a reliable coworker.
```

Comment:
I thought you were an oxford comma man? 🤣 
```suggestion
Brown uses the [BRAVING acronym](https://www.linkedin.com/pulse/braving-trust-brené-brown/) to refer to the elements of Trust. Out of these, the ones that jump out to me for saying no are boundaries, reliability, and integrity. To meaningfully trust others on these axes, you cannot be someone who says yes to every request. In doing so, you maintain no boundaries, professionally or personally, and will not be a reliable coworker.
```

Reply:
:trollface: